source/materials/OpticalMaterialProperties.cc

```diff
@@ -535,7 +596,7 @@ namespace opticalprops {
 
 
 
-  /// Gaseous Xenon ///
+  /// Gaseous xenon ///
   G4MaterialPropertiesTable* GXe(G4double pressure,
                                 G4double temperature,
```

Comment:
This gives an unused parameter warning, do we want to keep it or maybe add a dummy function as a placeholder?